transform/lambda_calculate_issue_score.py

```
@@ -0,0 +1,229 @@
+    # PostgreSQL 연결 설정 (실제 값으로 변경)
+    conn = psycopg2.connect(
+        dbname= conf.RDS_PROPERTY["dbname"], 
+        user= conf.RDS_PROPERTY["user"], 
+        password= conf.RDS_PROPERTY["password"],
+        host= conf.RDS_PROPERTY["url"],
+        port= conf.RDS_PROPERTY["port"], 
+    )
+    cur = conn.cursor()
+
+    # df_scaled의 각 행에 대해 issue_score 업데이트
+    for idx, row in df_scaled.iterrows():
```

Ruff (0.8.2)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)